src/components/ContentHighlights/data/constants.js

```diff
@@ -28,7 +40,19 @@ export const FOOTER_TEXT_BY_CONTENT_TYPE = {
   learnerpathway: 'Pathway',
 };
 
-// Test Data for Content Highlights
+// High Card logic for footer text
+export const getContentHighlightCardFooter = (price, contentType) => {
```

Comment:
nit: This isn't so much a constant, but a utility function. Would it make sense to move into a `utils.js` file? Perhaps also consider `export const getContentHighlightCardFooter = ({ price, contentType }) => { ... };`